### ISSUE TITLE
Add a shortcut to "Erase Terrain" tool

### ISSUE THREAD

**Issue by Luis-Tavares** [mapeditor/tiled#3126]

**Comment by bjorn** [comment]
The fastest way to switch to "Erase Terrain" mode is usually to right-click a location without terrain.

Do you have a suggestion for a shortcut? I'm thinking it would be nice if we could erase without changing the currently selected terrain, by modifier, but all modifiers are taken (Ctrl for larger brush, Shift for lines and Alt for 180-degree-symmetry). Potentially it would make sense to switch some of those to a mode that can be toggled, freeing up space for something as common as erasing.

**Comment by Luis-Tavares** [comment]
Yeah! At first, I thought about using "E" as a shortcut, but it might cause some conflicts with the common eraser tool.
Replacing the Alt modifier is a great idea indeed. Most art software (Aseprite, Photoshop, Clip Studio, etc) uses a toggle button for symmetry, so I don't see a problem changing that one.
If won't take so much work, you can also let the user change these modifiers on the "keyboard shortcuts" menu.

**Comment by bjorn** [comment]
> If won't take so much work, you can also let the user change these modifiers on the "keyboard shortcuts" menu.

Right, I think this is tricky since modifiers are very context-sensitive. They mean a lot of different things, so such a configuration would need to be done for each tool separately and then it will be nearly impossible to make sure there are no conflicts.

One possible way to do such a configuration without potential conflicts, would be to start at the modifier and then allow the user to select what it should do in a given context, rather than to present the action and then to allow selecting a modifier (as is currently done for keyboard shortcuts).
